desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -23,7 +26,9 @@ module.exports = class ArtworkAuctionView extends Backbone.View
     'change .js-artwork-auction-max-bid': 'setMaxBid'
 
   initialize: ({ @data }) -> #
-    setInterval before(10, @updateBidLabel), 1000
+    # Defer until next tick. This really just gets the sequence right for
+    # testing to work. This doesn't feel like a great justification, though.
+    setTimeout (=> @updateBidLabel LOT_STANDING_MAX_POLLS), 0
```

Comment:
The before hook in the tests uses a `__set__` mechanism I'm not sure I understand to affect the value of `AUCTION` in the scope of the class, but this only takes effect after instantiation of the class. I work around this by deferring to next tick. Open to better suggestions.